docs/smart-agent-quickstart.md

```diff
@@ -1,132 +1,311 @@
 
-In the referenced example agent.yaml configuration files below, the default
-location for the token file is used.
+* To [add a new monitor](#Monitors).
+* For [Common configuration options](https://docs.signalfx.com/en/latest/integrations/agent/monitor-config.html).
+* For [Windows monitor configurations](https://docs.signalfx.com/en/latest/integrations/agent/windows.html).
+* For [Linux monitor configurations](https://docs.signalfx.com/en/latest/integrations/agent/monitor-config.html)
 
-- [Linux Default Configuration File](https://github.com/signalfx/signalfx-agent/blob/master/packaging/etc/agent.yaml)
 
-- [Windows Default Configuration File](https://github.com/signalfx/signalfx-agent/blob/master/packaging/win/agent.yaml)
+### Configure optional metrics for your monitors. 
 
-##### Configure your endpoints
+For metric configuration for your monitor, see individual Windows or Linux monitor pages from the lists directly above or see the Integrations tab in the SignalFx application for Monitor Services.
 
-By default, the Smart Agent will send data to the `us0` realm.
-If you are not in this realm, you will need to explicitly set the
-`signalFxRealm` option in your config like this:
 
-```
-signalFxRealm: YOUR_SIGNALFX_REALM
-```
 
-To determine if you are in a different realm and need to
-explicitly set the endpoints, check your profile page in the SignalFx
-web application.
+## Add a New Observer 
 
-If you want to explicitly set the ingest, API server, and trace endpoint URLs,
-you can set them individually like so:
+See [Observers](#Observers).
 
-```
-ingestUrl: "https://ingest.YOUR_SIGNALFX_REALM.signalfx.com"
-apiUrl: "https://api.YOUR_SIGNALFX_REALM.signalfx.com"
-traceEndpointUrl: "https://ingest.YOUR_SIGNALFX_REALM.signalfx.com/v1/trace"
-```
 
-They will default to the endpoints for the realm configured in `signalFxRealm`
-if not set.
 
-You can add more [monitors](./monitor-config.md) and configure them as appropriate.
+## Explore Dashboards to display and compare data from various sources. 
 
-##### Example of adding a new monitor
+See [Built-In Dashboards and Charts](https://docs.signalfx.com/en/latest/getting-started/built-in-content/built-in-dashboards.html). 
 
-To start collecting apache metrics, you would add the [apache monitor](./monitors/collectd-apache.md) to the agent.yaml file.
-Your monitor list would then look similar to this:
+If you plan to create your own [custom dashboards](https://docs.signalfx.com/en/latest/dashboards/dashboard-basics.html#custom-dashboard-groups).
 
-```
+See best practices for [Better Dashboards](https://docs.signalfx.com/en/latest/reference/best-practices/better-dashboards.html). 
+
+
+
+###To learn more about how your data is presented in SignalFx, see the [15-Minute SingnalFx Quick Start](https://docs.signalfx.com/en/latest/getting-started/quick-start.html). 
+
+
+
+## Additional material 
+
+### Monitors 
+
+You can add more monitors and configure them as appropriate.
```

Comment:
Why would we want to add more monitors? Should we explain that a limited set of smart agent monitors are configured by default in the config file and that many more are available (see integrations tiles?)?

Reply:
Done.